scripts/variantstore/wdl/GvsCallsetCost.wdl

```diff
@@ -143,10 +143,10 @@ task ReadCostObservabilityTable {
         String cloud_sdk_docker
     }
     command <<<
-        bq --apilog=false query --project_id='~{project_id}' --format=prettyjson --use_legacy_sql=false \
-            "SELECT step, event_key, round(sum(event_bytes) / (1024*1024*1024), 2) AS sum_event_gibibytes \
-                FROM \`~{project_id}.~{dataset_name}.cost_observability\` \
-                WHERE call_set_identifier = '~{call_set_identifier}' GROUP BY step, event_key ORDER BY step" \
+        bq --apilog=false query --project_id=~{project_id} --format=prettyjson --use_legacy_sql=false \
+            'SELECT step, event_key, round(sum(event_bytes) / (1024*1024*1024), 2) AS sum_event_gibibytes \
+                FROM `~{project_id}.~{dataset_name}.cost_observability` \
+                    WHERE call_set_identifier = "~{call_set_identifier}" GROUP BY step, event_key ORDER BY step' \
```

Comment:
This one should be fine despite the single quotes since `~{call_set_identifier}` is a WDL variable.

Reply:
succeeding run of GvsCallsetCost.wdl on `rsa_vs_458` branch here: https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/24928de9-299a-42bd-a197-344ec7655e40